app/services/moves/finder.rb

```diff
@@ -59,7 +59,9 @@ def apply_filters(scope)
     end
 
     def split_params(name)
-      filter_params[name]&.split(',')
+      return if filter_params[name].blank?
```

Comment:
I'd prefer that we have consistent behaviour with this across all of the endpoints that support a comma delimited list - so either we don't do this anywhere, or it should also be done in the corresponding `split_params` method in `Locations::Finder`, `Allocations::Finder` and `People::Finder`.

I'm also not entirely clear why the front end can't just omit empty filters in the request and then no changes would be necessary? 🤔

Reply:
sure I've added them, we're going to test this, and if all is well I'll add all the necessary tests.

I think its mostly a bug/unintentional behaviour on our side, if I send a filter with an empty value I expect records to come back which have this value as empty rather than none.

Reply:
Okay, that's cool 👍